connector/connect/client/jvm/pom.xml

```diff
@@ -47,6 +47,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <!--TODO: fix the dependency once the catalyst refactoring is done-->
```

Comment:
Hi, @hvanhovell . 

I believe this is not a release blocker? Do we have a JIRA to track this?

Also, cc @xinrong-meng

Reply:
This is not a release blocker for Spark. But it is nice to be fixed as soon as the catalyst refactoring is complete.